arduino_code/openflexure_nano_motor_controller/openflexure_nano_motor_controller.ino

```diff
@@ -363,6 +567,19 @@ void loop() {
       #elif defined ADAFRUIT_ADS1115
       Serial.println(F("Light Sensor: ADS1115"));
       #endif
+      #if defined(ENDSTOPS_MIN)||defined(ENDSTOPS_MAX)
```

Comment:
Should there be a ||defined(ENDSTOPS_SOFT) ?

Reply:
Soft endstops currently only work with one set of physical endstops, so that is intentional (`ENDSTOPS_SOFT` is defined inside `defined(ENDSTOPS_MIN) || defined(ENDSTOPS_MAX)